napalm_ios/ios.py

```diff
@@ -919,91 +919,49 @@ def get_interfaces(self):
         output = self._send_command(command)
-        for line in output.splitlines():
-            if 'Interface' in line and 'Status' in line:
-                continue
-            fields = line.split()
-            """
-            router#sh ip interface brief
-            Interface                  IP-Address      OK? Method Status                Protocol
-            FastEthernet8              10.65.43.169    YES DHCP   up                    up
-            GigabitEthernet0           unassigned      YES NVRAM  administratively down down
-            Loopback234                unassigned      YES unset  up                    up
-            Loopback555                unassigned      YES unset  up                    up
-            NVI0                       unassigned      YES unset  administratively down down
-            Tunnel0                    10.63.100.9     YES NVRAM  up                    up
-            Tunnel1                    10.63.101.9     YES NVRAM  up                    up
-            Vlan1                      unassigned      YES unset  up                    up
-            Vlan100                    10.40.0.1       YES NVRAM  up                    up
-            Vlan200                    10.63.176.57    YES NVRAM  up                    up
-            Wlan-GigabitEthernet0      unassigned      YES unset  up                    up
-            wlan-ap0                   10.40.0.1       YES unset  up                    up
-            """
-
-            # Check for administratively down
-            if len(fields) == 6:
-                interface, ip_address, ok, method, status, protocol = fields
-            elif len(fields) == 7:
-                # Administratively down is two fields in the output for status
-                interface, ip_address, ok, method, status, status2, protocol = fields
-            else:
-                raise ValueError(u"Unexpected Response from the device")
-
-            status = status.lower()
-            protocol = protocol.lower()
-            if 'admin' in status:
-                is_enabled = False
-            else:
-                is_enabled = True
-            is_up = bool('up' in protocol)
-            interface_list[interface] = {
-                'is_up': is_up,
-                'is_enabled': is_enabled,
-                'last_flapped': last_flapped,
-            }
 
-        for interface in interface_list:
-            show_command = "show interface {0}".format(interface)
-            interface_output = self._send_command(show_command)
-            try:
-                # description filter
-                description = re.search(r"  Description: (.+)", interface_output)
-                interface_list[interface]['description'] = description.group(1).strip('\r')
-            except AttributeError:
-                interface_list[interface]['description'] = u'N/A'
+        interface = is_enabled = is_up = description =  mac_address = speed = speedformat = ''
```

Comment:
Wow.
If you really want to do this, at least take `is_up` and `speed` away and default them to bool, respectively int.